packages/orocksdb/orocksdb.0.2.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "Jan Doms <jan.doms@openvstorage.com>"
+authors: "Jan Doms <jan.doms@openvstorage.com>"
+homepage: "https://github.com/domsj/orocksdb"
+bug-reports: "https://github.com/domsj/orocksdb"
+license: "MIT"
+dev-repo: "https://github.com/domsj/orocksdb.git"
+build: [
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "rocks"]
+depends: [
+  "ocamlfind" {build}
+  "ctypes" { >= "0.4.0" }
+  "ctypes-foreign" { >= "0.4.0"}
+  "depext" {build}
```

Comment:
Is the `depext` dependency really needed for build? It should autoinstall if invoked (when using OPAM 1.2.2 or higher)